app/main.py

```diff
@@ -1,3 +1,77 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+
+from app.customer import Customer
+from app.shop import Shop
+from app.car import fuel_price, drive_to_shop, drive_to_home
+
+
+def shop_trip() -> None:
+    with open("config.json") as file:
```

Comment:
You should derive the directory name and join it with the `config.json` name to build the correct path to a file. Because of the incorrect file path, tests cannot pass.